app/controllers/sitemap_controller.rb

```diff
@@ -0,0 +1,66 @@
+class SitemapController < ApplicationController
+  skip_before_action :store_current_location
+  skip_before_action :load_objects_from_params
+  skip_before_action :load_html_blocks
+  skip_around_action :switch_locale
+
+  def index
+    respond_to do |format|
+      format.xml do
+        @collections = Collection.where(restricted: false, is_active: true)
```

Comment:
We do not want to restrict the sitemap to active collections.  Inactive collections are read-only and we want them to be crawlable.